openapi.yaml

```diff
@@ -0,0 +1,254 @@
+openapi: "3.0.0"
+info:
+  version: 0.1.0
+  title: Fic.AI Signals
+servers:
+  - url: https://fic.ai/v1
+paths:
+  /accounts:
+    post:
+      summary: Register a new account.
+      operationId: createAccount
+      tags:
+        - accounts
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateAccountRequest'
+      responses:
+        '201':
+          description: Created.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/EmptyObject"
+        '400':
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+  /sessions:
+    post:
+      summary: Log in to an existing account.
+      operationId: logIn
```

Comment:
Should we make operation ids match the method names in the code?

Reply:
How do you feel about settling on
* `create_account`
* `create_session`
* `get_signals`
* `patch_signals`

Trying to remember that the "accounts" url is maps to "user" in the backend and that "log_in" is spelled "session" In the front end isn't stellar.

I'm not in love with the `/signals` endpoint returning a struct named `Tags` either, particularly as in a later commit I add a `/tags` endpoint which I spelled `JustTags` for expediency at the time :p 

If there aren't objections I'll rename the `"user"` table to `account` while I'm at it so we can stop quoting it everywhere.

Reply:
Yeah, making the names be consistent across the project sounds like a good idea.

Reply:
Gave it a pass to cleanup naming and rearrange a few things, please let me know if there's anything outstanding you see.